README.ML

```diff
@@ -1,18 +1,15 @@
 Student: Pozdnikin Mykhailo
-Group: CE-47
```

Comment:
Why do you upload a binary files?

Reply:
This files are not binaries, but I want to push them into PR related to lab 2. This files are schematic files which can be open with fritzing application on desktop.